src/AvailabilityManager.php

```diff
@@ -32,16 +32,34 @@ public function getCheckers() {
    * {@inheritdoc}
    */
   public function check(PurchasableEntityInterface $entity, $quantity, Context $context) {
+    $min = .0000001;
+    $max = 9999999;
```

Comment:
I remember how someone discribed a use case where several millions were a common quantity in cart. I don't remember the details, but the learning: never assume 9999999 .. is absurdly large ;). Since than I use PHP_INT_MAX for such cases.